tpm2/tpm2.go

```diff
@@ -667,14 +682,14 @@ func decodeQuote(in []byte) ([]byte, []byte, error) {
 // Returns attestation data and the signature.
 //
 // Note: currently only RSA signatures are supported.
-func Quote(rw io.ReadWriter, signingHandle tpmutil.Handle, parentPassword, ownerPassword string, toQuote []byte, sel PCRSelection, sigAlg Algorithm) ([]byte, []byte, error) {
+func Quote(rw io.ReadWriter, signingHandle tpmutil.Handle, parentPassword, ownerPassword string, toQuote []byte, sel PCRSelection, sigAlg Algorithm) ([]byte, Signature, error) {
```

Comment:
Why return Signature instead of *Signature? If the latter, then this code (and similar calls) could return nil in the error case instead of a zero Signature struct.

Reply:
Was following pattern with Public, but agree - this make it cleaner. Done